### ISSUE TITLE
Lib load problems on suse

### ISSUE THREAD

**Issue by geosone** [onlime/sendmail-wrapper#1]
www:/etc/php5/cli # php -v
PHP 5.4.20 (cli) 
Copyright (c) 1997-2013 The PHP Group
Zend Engine v2.4.0, Copyright (c) 1998-2013 Zend Technologies
www:/etc/php5/cli # php -m
[PHP Modules]
Core
ctype
date
dom
ereg
filter
hash
iconv
json
libxml
mbstring
mhash
mysql
mysqli
mysqlnd
pcre
PDO
pdo_mysql
pdo_sqlite
Reflection
session
SimpleXML
SPL
sqlite3
standard
tokenizer
xml
xmlreader
xmlwriter

[Zend Modules]

www:/etc/php5/cli # echo "Subject: test" | sendmail-wrapper -v test@test.at Example

Fatal error: Class 'PDO' not found in /opt/sendmail-wrapper/app/SendmailThrottle.php on line 51

Fatal error: Call to undefined function mb_strlen() in /opt/sendmail-wrapper/app/SendmailWrapper.php on line 198

**Comment by RyanNutt** [comment]
I was getting the same PDO error, but not the mb_strlen.

Not sure that this is the fix, but what worked for me was to take out the -n in the hash bang at the top of sendmail-throttle.php. -n tells PHP not to load any .ini file and it's in that .ini file that the PDO extension gets loaded. Not sure why you wouldn't want to load an ini file at that point. Maybe @onlime can chime in with the importance of that.

**Comment by onlime** [comment]
@RyanNutt I have finally fixed that in f46f4c5. You were absolutely right, `php -n` was not necessary and prevented the wrapper from running on most systems. I was always running self-compiled PHP with most extensions compiled-in, so it didn't make any difference for me as all extensions were loaded. But if you use php packages from DEB.SURY.ORG or any other distro, modules are probably loaded via php.ini (or additional .ini files). So `php -n` would break it all!

AFAIK my idea at that time was: hey, let's be careful and not step into an endless loop, when `sendmail_path = /path/to/your/sendmail-wrapper.php` is set in CLI (php-cli's php.ini) as well. But hey, at that same time I forgot that I have actually made this configurable in config.ini:

```ini
[wrapper]
sendmailCmd = "/usr/sbin/sendmail -t -i"
```
so no matter what is set in php-cli's `sendmail_path`, I am passing the email anyway to the sendmail binary. Stupid me!
